.github/workflows/check.yml

```diff
@@ -114,6 +114,8 @@ jobs:
         run: |
           # shellcheck disable=SC2086
           cargo +${{ matrix.rust-toolchain }} llvm-cov nextest $BUILD_TYPE --features ci --no-fail-fast --lcov --output-path lcov.info
+          # shellcheck disable=SC2086
+          cargo +${{ matrix.rust-toolchain }} llvm-cov nextest $BUILD_TYPE --features ci,disable-encryption --no-fail-fast --lcov --output-path lcov-de.info
```

Comment:
So, looking again, what purpose does this serve?  To test that we have good coverage when encryption is disabled?  That seems like a non-goal to me.  Particularly since the run that disables encryption is merged with the regular run.

If the goal is to determine how much coverage our fuzzing tests have, then we should be running coverage for just the fuzzing runs.

Reply:
Happy to take this out